google/resource_google_project_organization_policy_test.go

```diff
@@ -0,0 +1,321 @@
+package google
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
+func TestAccProjectOrganizationPolicy_boolean(t *testing.T) {
+	t.Parallel()
+
+	folder := acctest.RandomWithPrefix("tf-test")
```

Comment:
Looks like a lot of this test was copied directly from the "folder" test and a fair number of "folder" references stuck behind. Mind cleaning those up?